ray-operator/config/samples/ray-cluster.overwrite-command.yaml

```diff
@@ -0,0 +1,60 @@
+# This example config is used to describe how does the annotation "ray.io/overwrite-container-cmd" work.
+# See kuberay/docs/guidance/pod-command.md for more details.
+apiVersion: ray.io/v1
+kind: RayCluster
+metadata:
+  annotations:
+    ray.io/overwrite-container-cmd: "true"
+  name: raycluster-overwrite-cmd
+spec:
+  rayVersion: '2.8.0' # should match the Ray version in the image of the containers
+  # Ray head Pod template
+  headGroupSpec:
+    rayStartParams: {}
+    # Pod template
+    template:
+      spec:
+        containers:
+        - name: ray-head
+          image: rayproject/ray:2.8.0
+          resources:
+            limits:
+              cpu: 1
+              memory: 2Gi
+            requests:
+              cpu: 500m
+              memory: 2Gi
+          ports:
+          - containerPort: 6379
+            name: gcs-server
+          - containerPort: 8265 # Ray dashboard
+            name: dashboard
+          - containerPort: 10001
+            name: client
+          # Because the annotation "ray.io/overwrite-container-cmd" is set to "true",
+          # KubeRay will overwrite the generated container command with `command` and
+          # `args` in the following. Hence, you need to specify the `ulimit` command
+          # by yourself to avoid Ray scalability issues.
```

Comment:
I think it will be helpful for the user if we remind them what $KUBERAY_GEN_RAY_START_CMD means here, even though you already described it in the code.